nix/tests/lanzaboote.nix

```
@@ -6,6 +6,9 @@ let
   inherit (pkgs) lib system;
   defaultTimeout = 5 * 60; # = 5 minutes
 
+  inherit (pkgs.hostPlatform) efiArch;
```

I think this should use the `hostPlatform` attribute from `stdenv`. I remember having problems with this otherwise but I don't remember the exact reason.

```suggestion
  inherit (pkgs.stdenv.hostPlatform) efiArch;
```